lua/starfall/libs_sh/mesh.lua

```diff
@@ -762,6 +775,191 @@ if CLIENT then
 		end
 	end
 
+	--- Starts a new dynamic mesh. If an Mesh object is passed, it will use that mesh instead
+	-- @param mesh Optional Mesh object (Default nil), mesh to use
```

Comment:
I dont like the description, because nil isnt a default, its just allowed value to not use any.